lib/cordial/products.rb

```diff
@@ -31,5 +31,32 @@ class Products
     def self.find(product_id:)
       client.get("/products/#{product_id}")
     end
+
+    # Create a new product.
+    #
+    # If the product already exists it will be updated.
+    # @example Usage.
+    #  Cordial::Products.create(
+    #    product_id: 1,
+    #    product_name: 'Test Product',
+    #    price: 99.99,
+    #    variants: [{
+    #      sku: '123',
+    #      attr: {
+    #        color: 'red',
+    #        size: 'L'
+    #      }
+    #    }]
+    #  )
+    def self.create(product_id:, product_name:, price:, variants:)
```

Comment:
What do you think about removing the product prefixes inside this file? 

`id:, :name, :price...`